src/computability/turing_machine.lean

```diff
@@ -745,6 +745,7 @@ begin
     λ a b h, of_to_bool_true $ (congr_fun h b).trans $ to_bool_tt rfl⟩,
   let H := (F.to_embedding.trans G).trans
     (equiv.vector_equiv_fin _ _).symm.to_embedding,
+  classical,
```

Comment:
Why does this become necessary?

Reply:
Because `set_value` is now computable but needs some `decidable` assumptions.

Reply:
OK, thanks.